pkg/kademlia/kademlia.go

```diff
@@ -275,16 +277,23 @@ func (k *Kademlia) FetchInfo(ctx context.Context, node pb.Node) (_ *pb.InfoRespo
 	return info, nil
 }
 
-// FindNode looks up the provided NodeID first in the local Node, and if it is not found
-// begins searching the network for the NodeID. Returns and error if node was not found
+// FindNode wraps a call to FindNodeWithVouchers with no vouchers
 func (k *Kademlia) FindNode(ctx context.Context, nodeID storj.NodeID) (_ pb.Node, err error) {
+	defer mon.Task()(&ctx)(&err)
+	return k.FindNodeWithVouchers(ctx, nodeID, nil, nil)
```

Comment:
If vouchers are nil, how are you going to send vouchers to other nodes. Is something missing here?

Reply:
The only time we want to send vouchers is during kademlia refresh, where we call `FindNodeWithVouchers`
```
func (k *Kademlia) refresh(ctx context.Context, threshold time.Duration) (err error) {
	...		
       _, _ = k.FindNodeWithVouchers(ctx, rID, vrs, &self) // ignore node not found
	...	
}
```